### PR TITLE
Added Dutch translation

### PR DESCRIPTION
**Pull request by Glennmen**
I added the Dutch translations for this library.

A couple comments:
* The library doesn't support plural, for example:
  * 1 minute -> 2 minutes
  * 1 minuut -> 2 minuten
* ~~Even though the library has multiple translations there is no way to easily use it in the library, you still need to copy/paste the array. `->setTranslation(string $language_code)` would be extremely useful.~~ My mistake, this part is handled by Laravel.

### CODE REVIEW

File: src/resources/lang/nl/read-time.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+return [
+    'reads_left_to_right' => true,
+    'min' => 'min.',
```
Comment (mtownsend5512) [inline src/resources/lang/nl/read-time.php]: Can you remove the '.' character, please

File: src/resources/lang/nl/read-time.php [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    'minute' => 'minuut',
+    'sec' => 'sec.',
```
Comment (mtownsend5512) [inline src/resources/lang/nl/read-time.php]: Can you remove the '.' character, please

File: src/resources/lang/nl/read-time.php (hunk 1 above)
Comment (Glennmen) [reply src/resources/lang/nl/read-time.php]: Done

File: src/resources/lang/nl/read-time.php (hunk 2 above)
Comment (Glennmen) [reply src/resources/lang/nl/read-time.php]: Done

**Pull request merged**